### ISSUE TITLE
[4.0.0-beta.15] ion-img does not work correctly on vivo android7.1

### ISSUE THREAD

**Issue by jytx** [ionic-team/ionic#16337]
# Bug Report

**Ionic Info**

```
Ionic:

   ionic (Ionic CLI)             : 4.3.1 (/usr/local/lib/node_modules/ionic)
   Ionic Framework               : @ionic/angular 4.0.0-beta.7
   @angular-devkit/build-angular : 0.7.5
   @angular-devkit/schematics    : 0.7.5
   @angular/cli                  : 6.1.5
   @ionic/angular-toolkit        : not installed

Cordova:

   cordova (Cordova CLI) : 8.0.0
   Cordova Platforms     : android 7.0.0
   Cordova Plugins       : cordova-plugin-ionic-keyboard 2.1.2, cordova-plugin-ionic-webview 2.1.0, (and 3 other plugins)

System:

   ios-deploy : 1.9.2
   NodeJS     : v10.13.0 (/usr/local/bin/node)
   npm        : 6.4.1
   OS         : macOS
   Xcode      : Xcode 10.0 Build version 10A255
```

**Describe the Bug**
when i use ion-img component it can't render on my device : vivo android7.1.1 ,but it can work well on my other device
i open chrome://inspect to inspect my app to debug the application , i find that <img> label doesn't have src attribute when i edit the html ,but when i add src attribute,then img will be shown

**Steps to Reproduce**
1. add <ion-img> on my page
```
<ion-img src="http://t2.hddhhn.com/uploads/tu/20150700/v45jx3rpefz.jpg"></ion-img>
```
2. add img on my page 
```
<img src="http://t2.hddhhn.com/uploads/tu/20150700/v45jx3rpefz.jpg" />
```
3. execute then command : ionic cordova run andorid  
4. see the result on my device

**Related Code**

```
<ion-img src="http://t2.hddhhn.com/uploads/tu/20150700/v45jx3rpefz.jpg"></ion-img>
<img src="http://t2.hddhhn.com/uploads/tu/20150700/v45jx3rpefz.jpg" />
```

**Expected Behavior**
ion-img and img will all be ok
**Additional Context**
![image](https://user-images.githubusercontent.com/9834906/48605558-0ca20880-e9b8-11e8-8a41-45d43a284af0.png)

**Comment by pauloiotech** [comment]
**Same problem happens to me**

```
Ionic:

   ionic (Ionic CLI)             : 4.4.0 (C:\Users\Paulo\AppData\Roaming\npm\node_modules\ionic)
   Ionic Framework               : @ionic/angular 4.0.0-beta.16
   @angular-devkit/build-angular : 0.10.7
   @angular-devkit/schematics    : 7.0.7
   @angular/cli                  : 7.0.7
   @ionic/angular-toolkit        : 1.2.0

Cordova:

   cordova (Cordova CLI) : 8.1.2 (cordova-lib@8.1.1)
   Cordova Platforms     : android 7.1.4
   Cordova Plugins       : cordova-plugin-ionic-keyboard 2.1.3, cordova-plugin-ionic-webview 2.2.5, (and 5 other plugins)

System:

   Android SDK Tools : 26.1.1 (C:\Users\Paulo\AppData\Local\Android\Sdk)
   NodeJS            : v10.13.0 (C:\Program Files\nodejs\node.exe)
   npm               : 6.4.1
   OS                : Windows 10
```

**Comment by chrisbinnefeld** [comment]
I'm can also verify this bug with `4.0.0-rc.0`. 

On newer WebViews (eg. 61.0.3163.98) all is working as expected. On WebViews with version <= 52.0.2743.100 the img won't be displayed. I used a fresh blank start project for testing.

![image](https://user-images.githubusercontent.com/36038291/50348128-2c3ccb80-0538-11e9-9911-102a5df2c31e.png)

> Ionic:
> 
>    ionic (Ionic CLI)             : 4.6.0 (C:\Users\cbinnefeld\AppData\Roaming\npm\node_modules\ionic)
>    Ionic Framework               : @ionic/angular 4.0.0-rc.0
>    @angular-devkit/build-angular : 0.11.4
>    @angular-devkit/schematics    : 7.1.4
>    @angular/cli                  : 7.1.4
>    @ionic/angular-toolkit        : 1.2.1
> 
> Cordova:
> 
>    cordova (Cordova CLI) : 8.0.0
>    Cordova Platforms     : android 7.0.0
>    Cordova Plugins       : cordova-plugin-ionic-keyboard 2.1.3, cordova-plugin-ionic-webview 2.3.1, (and 4 other plugins)
> 
> System:
> 
>    Android SDK Tools : 26.1.1 (C:\Users\cbinnefeld\AppData\Local\Android\Sdk)
>    NodeJS            : v10.10.0 (D:\installs\nodejs\node.exe)
>    npm               : 6.4.1
>    OS                : Windows 10

**Comment by paulstelzer** [comment]
Still an issue in rc2, but I think it has to do with the used Android Webview version like chris mentioned

**Comment by keo9ren** [comment]
Any news on this one. Facing the same issue on Galaxy Tab 2 with Chrome 74.0.3729. 157 as WebView using @ionic/angular 4.4.0.

Ionic Info
`Ionic:

   ionic (Ionic CLI)             : 4.12.0 (C:\Users\Kehret\AppData\Roaming\nvm\v11.15.0\node_modules\ionic)
   Ionic Framework               : @ionic/angular 4.4.0
   @angular-devkit/build-angular : 0.13.9
   @angular-devkit/schematics    : 7.3.9
   @angular/cli                  : 7.3.9
   @ionic/angular-toolkit        : 1.5.1

Cordova:

   cordova (Cordova CLI) : 9.0.0 (cordova-lib@9.0.1)
   Cordova Platforms     : android 7.1.0
   Cordova Plugins       : cordova-plugin-ionic-keyboard 2.1.3, cordova-plugin-ionic-webview 4.0.1, (and 10 other plugins)

System:

   NodeJS : v11.15.0 (C:\Program Files\nodejs\node.exe)
   npm    : 6.7.0
   OS     : Windows 10
`

**Comment by liugogal** [comment]
same problem

**Comment by liugogal** [comment]
When can this problem be solved? Although native img tags can be used instead, I think it's necessary to deal with them.

**Comment by jamespsterling** [comment]
Having this issue as well on Ionic 4.11.2

**Comment by speckro** [comment]
I am also having this issue. I am using React and I want to use IonImg because of the prop hidden.

**Android System WebView: 55.0.2883.91**
Android version 6.0.1
Ionic CLI: 6.6.0
Ionic Framework: @ionic/react 5.0.7
Capacitor CLI: 2.0.1
@capacitor/core: 2.0.1
NodeJS: v12.16.1
npm: 6.13.4
OS: Windows 10

The important part is the WebView. After updating it to the latest version, the problem is gone. But this is not a solution. I cannot expect from clients to update the WebView. As I understood correctly, the correct WebView is automatically installed as part of capacitor as the documentation says "Also, Capacitor requires an Android WebView with Chrome version 50 or later. On Android 5 and 6, the Capacitor uses the System WebView. On Android 7+, Google Chrome is used."

**Comment by liamdebeasi** [comment]
Thanks for the issue. This has been resolved in Ionic Framework v5.2.0. Please upgrade to that version to receive this fix. For any other bugs, please open a new issue. Thank you!
